### ISSUE TITLE
nytimes.com

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#147]
### Basic Information

**Test link:** `https://www.nytimes.com/2018/09/29/us/black-sisterhood-in-the-shadow-of-the-brett-kavanaugh-hearings.html`
**Category:** anti-adblock
**Reported from:** United States
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36
**Extension:** Nano Defender 15.0.0.48

**Comment by jspenguin2017** [comment]
Fixed here: https://github.com/NanoMeow/QuickReports/issues/90
Fix verified working.

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://www.nytimes.com/2018/09/30/us/migrant-children-tent-city-texas.html`
**Category:** anti-adblock
**Reported from:** United States
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36
**Extension:** Nano Defender 15.0.0.50

### Additional Message

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://www.nytimes.com/2019/08/08/us/politics/soulcycle-equinox-boycott.html`
**Category:** other
**Reported from:** Australia
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:70.0) Gecko/20100101 Firefox/70.0
**Extension:** Nano Adblocker 1.0.0.123

### Additional Message

giant subscribe banner at the bottom covering a massive part of the page

**Comment by jspenguin2017** [comment]
Checked 7 articles, only saw a small banner once. Non-intrusive self-promotion is allowed, won't fix.

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://www.nytimes.com/2020/01/28/world/asia/china-coronavirus.html`
**Category:** breakage
**Reported from:** United States
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:73.0) Gecko/20100101 Firefox/73.0
**Extension:** Nano Defender Pro 15.0.0.180

### Additional Message

`Nano Contrib Filter - Placeholder Buster` is blocking `Here’s what you need to know:` info box with bad filter `nytimes.com##section[name="articleBody"] > div:not(.StoryBodyCompanionColumn):not([data-testid]):not(:last-of-type)`

**Comment by yourduskquibbles** [comment]
`Read Updates in Chinese: 武汉疫情每日汇总中文版` section also blocked by same filter

**Comment by llacb47** [comment]
@dandelionsprout @jspenguin2017

**Comment by jspenguin2017** [comment]
Contribution welcomed.

**Comment by jspenguin2017** [comment]
https://github.com/NanoAdblockerLab/NanoContrib/commit/817dac4b2619d2bbffa922a4435f24c1a7fc896b
